src/readDwarf.cpp

```diff
@@ -359,7 +359,7 @@ Location decodeLocation(const DWARF_Attribute& attr, const Location* frameBase,
 		}
 	}
 
-	assert(stackDepth > 0);
+	//assert(stackDepth > 0);

```

Comment:
Why are these asserts removed?

Reply:
I was hitting this and another assert while testing with a Golang CGO DLL.  I do not really know what is the problem being detected by this (and the other one I commented out) asserts.

I can roll these back as I have not touched this code.

Reply:
Probably better to keep the asserts, so it can be investigated if it comes up again.

Reply:
Removed the commit